mcs/class/corlib/corlib.dll.sources

```diff
@@ -142,6 +142,7 @@ System/TimeZoneInfo.Android.cs
 System/TimeZoneInfo.MonoTouch.cs
 System/TimeZoneInfo.Serialization.cs
 ../../build/common/MonoTODOAttribute.cs
+System/TypeIdentifier.cs
```

Comment:
Does that just mean the other `*.sources` files in this directory? Or are there other places that need to change?